frontend/packages/core/src/AppLayout/user.tsx

```diff
@@ -76,34 +128,52 @@ const UserInformation: React.FC = () => {
   }
 
   return (
-    <Grid container alignItems="center" justify="flex-end">
-      <div>{userId()}</div>
+    <Box>
       <UserPhoto
         ref={anchorRef}
         edge="end"
         aria-controls={open ? "account-options" : undefined}
         aria-haspopup="true"
         onClick={handleToggle}
       >
-        <Avatar>
-          <Initials variant="h6">{userId().slice(0, 2).toUpperCase()}</Initials>
-        </Avatar>
+        <UserAvatar />
       </UserPhoto>
-      <Popper open={open} anchorEl={anchorRef.current} role={undefined} transition>
+      <Popper
+        open={open}
+        anchorEl={anchorRef.current}
+        role={undefined}
+        transition
+        popperOptions={{
+          modifiers: {
+            offset: {
+              offset: "-116, -112",
+            },
+          },
+        }}
```

Comment:
We probably shouldn't inline style here but just for simplicity I think you can achieve the same thing with the following:
```suggestion
        placement="bottom-end"
        style={{
          padding: "0 12px",
          marginLeft: "12px"
        }}
```

The addition margin on the left is due to the `edge="end"` on the `UserPhoto`. Looking at the story, if you remove the grid wrapper around the user information the current logic breaks because the offset it hard coded. With these changes you can remove that wrapper and it'll still render with correct alignment when opening.

Reply:
thank you! this is so much better. fixed